tests/queries/0_stateless/02174_cte_scalar_cache.sql

```diff
@@ -0,0 +1,48 @@
+WITH
+    ( SELECT sleep(0.0001) FROM system.one ) as a1,
+    ( SELECT sleep(0.0001) FROM system.one ) as a2,
+    ( SELECT sleep(0.0001) FROM system.one ) as a3,
+    ( SELECT sleep(0.0001) FROM system.one ) as a4,
+    ( SELECT sleep(0.0001) FROM system.one ) as a5
```

Comment:
Should we maybe sleep for `0.1` sec and set `max_execution_time` to `0.2` or something like that?